px-spinner.html

```diff
@@ -76,34 +79,6 @@
                 return Px.Modernizr.smil;
               }
             }
-        },
-        /**
-         *
-         * This method checks whether the finished attribute is true,
-         * and if it is, hides the element. if it's false, it shows it
-         *
-         */
-        _finishedChanged: function(newValue, oldValue) {
-          (this.finished) ? this.hide() : this.show();
-        },
-
-        /**
-         * Shows the spinner
-         *
-         * @method show
-         */
-        show: function() {
```

Comment:
Should we keep these methods so this isn't an API breaking change? They should be able to just toggle the `finished` property I think.

Reply:
yeah good point. For some reason I was thinking these were only internal properties. I'll update them both.